src/index.tsx

```diff
@@ -39,6 +42,23 @@ const RouterWithInit = () => {
   useEffect(() => {
     store.dispatch(subscribeToConfig());
   }, []); // Empty-array means "run only once on init": https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects
+
+  // Initialize global subscription to firestore requests evaluation
+  useEffect(() => {
```

Comment:
This won't work once websocket host and port is dynamic (derived from config). Better put this in sagas and subscribe in response to config fetch success action. Remember that ports may change (e.g. when emulators are restarted) so you'd need to unsubscribe and resubscribe if that happens.

Reply:
We decided to not work on this code yet, since it will not be necessary until later on.